web/src/Web.App/AssetSrc/scss/main.scss

```diff
@@ -389,4 +389,20 @@ hr.govuk-section-break--print {
   @media print {
     opacity: 0;
   }
+}
+
+.app-resources {
+  margin-top: 1.5rem;
+  
+  @include govuk-media-query('tablet') {
+    margin-top: 0;
+  }
+}
+
+.app-resources-warning-spacing {
+  padding-top: 0;
+
+  @include govuk-media-query('tablet') {
+    padding-top: 0.625rem;
```

Comment:
nitpick: To prevent the 'magic number' here and above you should be able to import from [GDS](https://design-system.service.gov.uk/styles/spacing/) using the appropriate mixin. e.g. `govuk-spacing()`.